var/spack/repos/builtin/packages/variorum/package.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Variorum(CMakePackage):
+    """Variorum is a library providing vendor-neutral interfaces for
+       monitoring and controlling underlying hardware features.
+    """
+    homepage = "https://variorum.readthedocs.io"
+    git = "https://github.com/llnl/variorum.git"
+    url = "https://github.com/LLNL/variorum/archive/v0.1.0.tar.gz"
+
+    maintainers = ["slabasan", "rountree"]
+
+    version("0.1.0", tag="v0.1.0")
+
+    ############
+    # Variants #
+    ############
+    variant("shared", default=True, description="Build Variorum as shared lib")
+
+    # variants for language support
+    variant("python", default=True, description="Build Variorum Python support")
```

Comment:
The Python support doesn't have a Python dependency or any CMake flags?

Reply:
I don't believe so, should `python` still be a variant in this case?

Reply:
Probably not

Reply:
Ok removed this variant.